web/src/reducers/activities.js

```diff
@@ -256,6 +258,96 @@ const reducer = (state = initialState, action) => {
         },
         state
       );
+    case GET_APD_SUCCESS: {
+      const byId = {};
+      action.data.activities.forEach(a => {
+        byId[a.id] = {
+          id: a.id,
+          name: a.name,
+          types: ['HIT'], // TODO
+          descShort: '', // TODO
+          descLong: a.description,
+          altApproach: '', // TODO
+          costAllocateDesc: '', // TODO
```

Comment:
These 4 `TODO`s are partially represented in #554. After that one is done, we'll need to come back and update these.

Reply:
👍